omod/src/main/java/org/openmrs/module/appframework/rest/ExtensionResource.java

```diff
@@ -104,4 +112,28 @@ protected PageableResult doSearch(RequestContext context) {
         }
         return new NeedsPaging<Extension>(results, context);
     }
+
+    @Override
+    public Model getGETModel(Representation rep) {
+        ModelImpl modelImpl = (ModelImpl) super.getGETModel(rep);
+
+        if (rep instanceof DefaultRepresentation || rep instanceof FullRepresentation) {
```

Comment:
@dkayiwa okay I'll try explain. :)

See the below test:
https://github.com/openmrs/openmrs-module-appframework/blob/c3e2b6f229a13bd43018ac6adcf743903e59da8d/omod/src/test/java/org/openmrs/module/appframework/rest/ExtensionResourceTest.java#L28-L40

As you could see by the assert statements the default representation include all the attributes. But if I change this to:

 SimpleObject actual = new ExtensionResource().asRepresentation(extension, **Representation.FULL**); 

.. the returned response of the test only include `uuid`, `display`, and `links`. So if that's the correct response for FULL, we'd need edit this, because @f4ww4z changes consider responses of DEFAULT, and FULL representations are both same.

Reply:
I see, I just thought that FULL representation has more or the same properties than DEFAULT rep. I'll change this when @dkayiwa agrees.

Reply:
Yes FULL >= DEFAULT

Reply:
What is the best solution here @dkayiwa ? :)